server/admin/admin_server_test.go

```diff
@@ -318,10 +347,11 @@ func TestServer_CreateTree(t *testing.T) {
 	keySignatureMismatch.SignatureAlgorithm = sigpb.DigitallySigned_RSA
 
 	tests := []struct {
-		desc                           string
-		req                            *trillian.CreateTreeRequest
-		createErr                      error
-		commitErr, wantErr, wantCommit bool
+		desc                  string
+		req                   *trillian.CreateTreeRequest
+		createErr             error
+		commitErr, wantCommit bool
+		wantErr               string
```

Comment:
Asserting on error strings is likely to make tests brittle. Ideally, we'd use something more resilient to small changes, like error codes, but I that won't help if the codes are all the same. OTOH, it does make it easier to reason that the error is what you think it is, and not some problem.

WDYT?